trinity/server.py

```diff
@@ -344,6 +360,13 @@ def _make_syncer(self) -> LightChainSyncer:
             self.cancel_token,
         )
 
+    def _make_request_server(self) -> LightRequestServer:
+        return LightRequestServer(
+            self.headerdb,
+            cast(LESPeerPool, self.peer_pool),
```

Comment:
I'm surprised this `cast` is needed though I do remember there being some oddities with getting the types to be right with this.

Reply:
Found a way around it by making the `Server` into `Generic`